main.py

```diff
@@ -228,7 +228,11 @@ def main(league, time, standings, team, live):
         get_team_scores(team, time)
         return
 
-    get_scores(league, time)
+    if league is None:
+        for league in LEAGUE_IDS.keys():
```

Comment:
you dont need to loop over LEAGUE_IDS.keys()...you can just do:
for league in LEAGUE_IDS:
    ....
This will loop over the keys of the dictionary